CLAUDE.md

```
@@ -0,0 +1,110 @@
+npm run lint:fix     # auto-fix lint issues
+npm run format       # run Prettier on all files
+npm run format:check # check formatting without writing
+npm run typecheck    # TypeScript type-check all packages
+npm run test         # run Vitest unit tests
+npm run generate     # regenerate the apps table in README.md
+npm run generate -- --check  # verify README is up-to-date (used in CI)
+```
+
+## Repo Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)